### ISSUE TITLE
Parking Office Meeting - Date Friday Sept 27th 

### ISSUE THREAD

**Issue by nguyes44** [nguyes44/MacTransit#10]
**Date and Time** Friday Sept 27th, 1:30pm to 2:30pm

**Attendance**

- [x] Aysu Ozdal
- [x] Andrew Thompson
- [x] Arvind Shastri
- [x] George Yazji
- [x] Stanley Nguyen

**Agenda**

Priority:
- Run through and confirm proposed goals
- What resources are you able to provide? (route info, schedules, etc)
- Can we load a GPS module onto the busses?
- Will shuttle bus drivers be able to use their devices to select the bus route on the app? or would using external devices such as a mounted table be a possibility?
- Given that the goals are agreed upon, give a rough estimate of yearly costs to maintain **(maybe lets not mention cost right now as we are not sure)**
- How often will we be able to meet moving forward? 
- What is the parking office's budget? (this helps us know which solutions to explore)
- What are future goals for the parking office? (new routes, new stops, new functionality, etc.)

Lower Priority:
- What kind of privacy does the PO require for this project's code? Licenses include MIT, GPL, etc. Also IP, confidential info.
- Can we get in contact, through you, with a mailing list of commuters using transponders or parking often in off-campus parking lots? If so, can we use your platform to mail them a survey to gather information? (Do we need to contact the research board before doing this?)
- What are the odds this application will continue to be used afterwards? Who will be maintaining it?
- Opinion on stretch goals (data analysis, equal distance)
- Ask about other teams/projects being worked on

Action items after the meeting:

1.  Change documents to mention any new requirements/new stakeholders
2. Generate cost estimate
3. Get data from the PO such as analysis and drivers count

**Comment by arvindshastri** [comment]
- Part of the RnD campaign of the PO to create a transit-like app for the university
- Out of the three competing groups that have approached the PO, the best group has opportunity to sell their app / partnership to the shuttle bus company, Attridge
- The PO is looking for an app that works on Apple and Android, and gives you an idea of where the shuttle bus is on campus and how much time it'll take to arrive at your nearest stop
- The PO is thinking to include the 10 Bay and Ron Joyce shuttle buses alongside the regular parking lot buses to be tracked in this app
- The PO is working on branded McMaster transit and cobranded Attridge transit bus lines, as well as moving some of the buses to EVs in the future
- Dean will handle any barriers related to testing, is willing to give us access on campus to do testing with the buses (to add GPS modules, close off lanes to perform routes, etc.) and provide us further info about bus schedules, timings, etc. at a later date
- Dean loves the idea of being able to send out information for status updates, route updates, etc.
- Dean likes the idea of having a transit app that can track all the transit options on campus (GO Transit, FlixBus, HSR, shuttle buses)

- Bus drivers should know where the next bus is and what the best way to send them out on their route is.
- Dean: the main reason for buses coming on the route back to back is because when classes end and students are coming out, buses stop for pedestrian safety and this delay causes the buses to end up back to back along the way

- Q: Can/should bus drivers interact with the app?
    - Drivers currently take a mental count of how many people come on/off the bus at given stops during the day
    - The drivers input this information manually into MyChart/spreadsheets to gather data on monthly rider usage etc.
    - Dean likes the idea of incorporating a way for buses to do this data collection within the app itself.
    - This data can help us look into data analysis with enough collected information.
  
- Any budget approvals need to go through the PO director.
- Attridge will become our partner for budget discussions etc.
- Dean doesn't want to manage the app after we graduate. All this logistics discussion should be held with the company, concerning training them, giving them user documentation, etc.

- Dean will connect us to a web developer to create a survey to gather information from the public
- The data from the PO's survey will be collected and distributed to all three competing groups
- Proposes we start manually gathering data with a QR code at each bus stop (SurveyMonkey, etc.)

- We have to be prepared to show a high fidelity prototype to the company for feedback and direction
- When we're ready with a POC (which he hopes can be done within 6 to 8 weeks), all three groups will sit at a board meeting in front of the VP of Attridge and pitch. The business details will be proposed and further discussed at that point.

- One competing group is going to West Prince Hall and looking to take a small shuttle bus through that loop.

**Comment by thompa39** [comment]
- Will support us and let us use them as a testing ground
- 3rd group that has approached them
- Will introduce us to the shuttle bus company (Attridge)
  - Will meet with each group and see which one is best

What Dean wants:
  
- Works on Apple or Android
- Can look at app in real time to see where shuttle is
- Also have 10 Bay shuttle included, and Ron Joyce Center
  - Identify on a tab and see where Ron Joyce shuttle is
- Branded MacTransit, co-branded with Attridge
- Make sure we have access to do testing on campus with the buses
- Even if we don't end up selling to the company, we would be able to create the program

- If we need info such as routes, GPS modules
- Introduction to the VP of Attridge
- From the pitch perspective
- Testing and doing things on campus would go through Dean
- Likes to hear bus driver features
  - Tracking how many people get on/off, automatic data plotting and analysis

Budget would be difficult, let's look at what costs look like, trying to minimize costs since we are students
- We can get back to you on a better estimate

Any post-uni maintenance would be done through the company

Survey wouldn't be quick, because he is in between website admins
QR codes at stops

Pitch prototype, lets them know what direction we would be going in

Links in app to other transit (ex. GO, FlixBus, HSR)

Speak to advisors about whether to sell license or sell the project itself

8-10 months for the new buses (electric buses)

Dean will connect us with an IT person once they've been brought in
    
He will make arrangements for on-bus testing

They want the tracking so that buses can know where each other are

Proposed route going up to Les Prince Hall as part of the loop from Wes

**Comment by aysuozdal** [comment]
•	Introduce to the shuttle bus company
	◦	External company
	◦	Create an app for university
	◦	We would like to meet them
	◦	They pick the past (business)
	◦	Attridge transportation???

**Comment by george536** [comment]
Action items after the meeting:

**Comment by aysuozdal** [comment]
- Introduce to the shuttle bus company
    - External company
    - Create an app for university
    - We would like to 
    - They pick the past (business)
    - Attridge transportation???
- Something both work on ios/android
- Real time location
- Should be able to see from wherever
- Be able to identify the shuttles (there is different ones)
- Branded mac transit/ have the ability to brand that
- Still be able to complete the project even if we don't get picked
- Need to get authorisation from the company to get information
- Great to have the admin part/ pushing notifications feature
    - Traffic is also a parameter
- Counting ppl (drivers already keep metrics)
    - Should be able to log the data
    - He can show us the monthly data they get from the drivers
- He has total ppl/day/bus
- Graphs etc -- shows the number of ppl
    - Helps with the size of the bus
    - Maybe add another bus
    - Helpful for decision making
- Old school data collecting -- can have a new technology to that
    - More user friendly
- Maintenance: need to look at the costs -- potentially help with it
    - Further investigation
    - Bus company will become the partner
    - Licence maintenance will be dealt with them as well
- Create a survey once the website is done -- will be shared with all groups
- QR code survey -- before riding the bus
- The business room also looking into banking part???
- Min cost is preferrable
- Can add metrolinx -- go bus (to open up the other up), HSR, Flix Bus, busses that take off from mcmaster campus
    - Like an uber
- Selling licence needs to be talked with the supervisor as well
- They will need someone to maintenance
- 8-10 months EV busses coming up, all buses will be changed
    - Sustainability
    - Same bus company
- Will be in touch with the IT office
- Can't distract the bus driver
- Present to the parking office
    - Status updates
- Need to talk to the prof since there is a 3rd party involved
- Metrics, user friendly,
- Bus drives know how many ppl are there
    - Sometimes they get slowed down bc everyone wants to get in al at the same time
- Should keep the busses circulating, shouldn't really be behind each other
- Might have new busses that are rotating -- support multiple routes
- Data: they can share -- by november??
